pkg/db/drivers/etcd/etcd.go

```diff
@@ -260,6 +260,17 @@ func (c *Client) FindFileShareValue(k string, p *model.FileShareSpec) string {
 }
 
 func (c *Client) CreateFileShareAcl(ctx *c.Context, fshare *model.FileShareAclSpec) (*model.FileShareAclSpec, error) {
+	acls, err := c.ListFileSharesAcl(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, acl := range acls {
+		if acl.AccessTo == fshare.AccessTo {
+			log.Error("acl already exists for this ip: "+acl.AccessTo+". If you want to set new acl, first delete the existing one")
```

Comment:
You can create a string var and put this message, return the message rather than duplicating the error string line

Reply:
Done